wallet/src/main/java/com/android/identity_credential/wallet/WalletApplication.kt

```diff
@@ -200,17 +201,63 @@ class WalletApplication : Application() {
             getWalletApplicationInformation()
         }
 
-        // init TrustManagers
+        // init TrustManager for readers (used in consent dialog)
+        //
         readerTrustManager.addTrustPoint(
             displayName = "OWF Identity Credential Reader",
             certificateResourceId = R.raw.owf_identity_credential_reader_cert,
             displayIconResourceId = R.drawable.owf_identity_credential_reader_display_icon
         )
+        for ((name, certResourceId) in listOf(
+            Pair("Credence ID", R.raw.austroad_test_event_reader_credence_id),
```

Comment:
super nit: it's probably a bit more readable as "Credence Id" to R.raw.austroad_test_event_reader_credence_id, rather than using Pair

Reply:
Got it! Actually turns out we're not using `name` at all so just going to simplify it to just iterating over the resource IDs.